packages/fiori/src/types/MediaGalleryLayoutType.js

```diff
@@ -0,0 +1,48 @@
+import DataType from "@ui5/webcomponents-base/dist/types/DataType.js";
+
+/**
+ * @lends sap.ui.webcomponents.fiori.types.MediaGalleryLayoutType.prototype
+ * @public
+ */
+const MediaGalleryLayoutTypes = {
+
+	/**
```

Comment:
Just leaving this as topic for brief discussion, we have:
- MediaGalleryItemLayoutTypes - square and wide
- MediaGalleryLayoutTypes - auto, vertical, horizontal

Both enums used for property with name "layoutType", but contain too different options, could be a topic for improvement.